### ISSUE TITLE
Chocolatey still shows older v1.21.2 two days after release of v1.21.3

### ISSUE THREAD

**Issue by tyler36** [drud/ddev#4318]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Run a Diagnostic and Paste Link Here

Recently, DDEV version `1.21.3` was released (2 days ago).  I was able to upgrade my WSL install via brew.

On Win10, I use choco to manage my installs. When I check for updates, it said my current `1.21.2` is the latest. I check the [choco website](https://community.chocolatey.org/packages/ddev) which also reports `1.21.2` as the latest.

### Expected Behavior

I expect the choco to be the same as the current release, give or take a few hours for caches etc.

- [Choco website](https://community.chocolatey.org/packages/ddev) should display latest version
- [DDEV release](https://github.com/drud/ddev/releases)

### Actual Behavior

DDEV release reports `1.21.3`
Choco reports `1.21.2`

### Anything else?

Not sure we are moving away from Choco support or something is broken in our release process.

**Comment by rfay** [comment]
Thanks, sometimes they get way behind. They failed the first time due to their own api failing, and I manually re-pushed it yesterday. 

I can see it at the link but I don't think you can: https://community.chocolatey.org/packages/ddev/1.21.3

Apparently they're behind on scan testing.  I don't know of any way to make them go faster. Here's the gist of where they're at, https://gist.github.com/choco-bot/91354ba666fcb933cbd87e3ba2bfc4d7

If it takes too many days longer we'll go into their discord and ask.

In the meantime, it's fine to download the installer from https://github.com/drud/ddev/releases/tag/v1.21.3

<img width="1353" alt="Chocolatey_Software___ddev__Install__1_21_3" src="https://user-images.githubusercontent.com/112444/197084612-2b4ede63-b570-4f8b-a6f2-d44eb68f95b4.png">

**Comment by tyler36** [comment]
Thanks. I was able to to `1.21.3` page with the link you provided and the "Pending automated review" notice.

I use DDEV almost exclusively in WSL. Only logged the issue so we could track things if they were broken. :)

**Comment by rfay** [comment]
It *finally* came through this morning. https://community.chocolatey.org/packages/ddev/1.21.3

**Comment by rfay** [comment]
Reopening, as the actual manual push had problems, see https://github.com/drud/ddev/pull/4321

**Comment by tyler36** [comment]
Confirm. I'm seeing `1.21.3` appear in Powershell.

**Comment by tyler36** [comment]
Just tried to update and am getting an error:

```
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://github.com/ddev/releases/download/v1.21.3/ddev_windows_installer.v1.21.3.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
The upgrade of ddev was NOT successful.
```

**Comment by tyler36** [comment]
Attempting to access the EXE via the browser shows a blank screen with `Not Found`. 😢

**Comment by rfay** [comment]
I pushed 1.21.3.1 and I think it will probably succeed. https://community.chocolatey.org/packages/ddev/1.21.3.1

Not sure how the other one could have passed their verification testing.

**Comment by rfay** [comment]
https://community.chocolatey.org/packages/ddev/1.21.3.1 is approved and I was able to install using `choco upgrade ddev` to get ddev v1.21.3. Thanks for staying on top of this @tyler36

**Comment by tyler36** [comment]
Confirming. Was able to update this morning.

Just 1 point though, it shows the version as `1.21.3`, instead of `1.21.3.1`.  Is that righ?

```shell
❯ choco upgrade all -y
...
Upgraded:
 - ddev v1.21.3.1

❯ ddev -v
ddev version v1.21.3
```

**Comment by rfay** [comment]
Yes, the only actual release is v1.21.3, but Chocolatey won't let you push a new release, so I had to push a new patch-patch chocolatey release... which pointed to the original v1.21.3.
